packages/modeling/src/operations/booleans/scissionGeom3.js

```diff
@@ -0,0 +1,93 @@
+const vec3 = require('../../maths/vec3')
+const measureEpsilon = require('../../measurements/measureEpsilon')
+
+const geom3 = require('../../geometries/geom3')
+
+// returns array numerically sorted and duplicates removed
+const sortNb = (array) => array.sort((a, b) => a - b).filter((item, pos, ary) => !pos || item !== ary[pos - 1])
+
+const insertMapping = (map, point, polygonIndex) => {
+  const key = `${point}`
+  const mapping = map.get(key)
+  if (mapping === undefined) {
+    map.set(key, [polygonIndex])
+  } else {
+    mapping.push(polygonIndex)
+  }
+}
+
+const findMapping = (map, point) => {
+  const key = `${point}`
+  return map.get(key)
+}
+
+const scissionGeom3 = (geometry) => {
+  // construit table de correspondance entre polygones
+  // build polygons lookup table
+  const eps = measureEpsilon(geometry)
+  const polygons = geom3.toPolygons(geometry)
+  const pl = polygons.length
+
+  const polygonIndexesPerPoint = new Map()
+  const temp = vec3.create()
+  polygons.forEach((polygon, polygonIndex) => {
+    polygon.vertices.forEach((point) => {
+      insertMapping(polygonIndexesPerPoint, vec3.snap(temp, point, eps), polygonIndex)
+    })
+  })
+
+  const zz = polygons.map((polygon, polygonIndex) => {
+    const indexes = []
+    polygon.vertices.forEach((point) => {
+      indexes.push(...findMapping(polygonIndexesPerPoint, vec3.snap(temp, point, eps)))
```

Comment:
Maybe concat() is a better choice as there are limits on the size of arrays using '...'